New docs and examples (#84)
3scale-labs/authorino: docs/contributing.md

# Contributing to Authorino

## Open issues

> Start by checking the list of [issues](https://github.com/3scale-labs/authorino/issues) in GitHub. Sometimes your contribution may have been filed in the past or part of another ongoing conversation.

Reviewer: ```suggestion
Start by checking the list of [issues](https://github.com/3scale-labs/authorino/issues) in Github. Maybe your idea was discussed in the past or is part of an ongoing conversation.
```

> If you believe that your contribution is new, please open an issue before sending your pull-request.

Reviewer: Do we want to say something like" open an issue, describe your idea and start a conversation"?

Author: I hope it's better now.
